control_plane/roles/control_plane_common/tasks/fetch_security_inputs.yml

```diff
@@ -43,3 +43,41 @@
     success_msg: "{{ realm_success_msg }}"
     fail_msg: "{{ realm_fail_msg }}"
   tags: [ validate, security ]
+
+- name: Validate max_failures
+  assert:
+    that:
+      - max_failures | int == 3
```

Comment:
Abhishek, let's convert all the magic numbers in the variable.

Reply:
All numbers are updated as variables.